static/app/utils/groupCapabilities.tsx

```diff
@@ -0,0 +1,27 @@
+import {BaseGroup, IssueCategory, IssueCategoryCapabilities} from 'sentry/types/group';
+
+/**
+ * Defines what capabilities each category of issue supports
+ */
+const ISSUE_CATEGORY_CAPABILITIES: Record<IssueCategory, IssueCategoryCapabilities> = {
+  [IssueCategory.ERROR]: {
+    delete: true,
+    ignore: true,
+    merge: true,
+  },
+  [IssueCategory.PERFORMANCE]: {
+    delete: false,
+    ignore: false,
```

Comment:
Is ignoring not supported? For some reason I thought only delete and merge wouldn't work

Reply:
Nevermind this is answered above